wagtail/images/static_src/wagtailimages/scss/listing.scss

```diff
@@ -0,0 +1,49 @@
+@import '../../../../../client/scss/settings';
+@import '../../../../../client/scss/tools';
+
+@include media-breakpoint-up(sm) {
+  .fields {
+    max-width: 1000px;
+
+    .field-col {
+      padding-inline-start: 0;
+
+      .field {
+        display: flex;
+        flex-direction: column;
+
+        .field-content,
+        label {
+          width: auto;
+          max-width: 85%;
+          padding-top: 0;
+        }
+      }
+    }
+
+    > li {
+      padding-top: 0;
+    }
+
+    .tagfilter legend {
+      width: auto;
+    }
+  }
+}
+
+.listing.horiz {
```

Comment:
Is there a reason we want to do this here, rather than modifying the `_listing.scss` directly and being more consistent across other views?

Reply:
I didn't want to break some views. I can do it in `_listing.scss` and test how it goes.

Reply:
I think that would be good - presumably we want the same style for the image chooser, for example?

Reply:
Yes, that makes sense. I've updated it. (The image chooser seems to be more consistent with the listing view with the change).